cpp/benchmarks/copying/contiguous_split_benchmark.cu

```diff
@@ -34,10 +34,17 @@ void BM_contiguous_split_common(benchmark::State& state,
                                 int64_t bytes_total)
 {
   // generate splits
-  cudf::size_type split_stride = num_rows / num_splits;
   std::vector<cudf::size_type> splits;
-  for (int idx = 0; idx < num_rows; idx += split_stride) {
-    splits.push_back(std::min(idx + split_stride, static_cast<cudf::size_type>(num_rows)));
+  if (num_splits > 0) {
+    cudf::size_type const split_stride = num_rows / num_splits;
+    auto iter                          = thrust::make_counting_iterator(1);
```

Comment:
why does this start at 1? Should this be commented or a `constexpr`?

Reply:
Starts at 1 just so that the first split happens at a useful place.  I'll add a comment.